README.md

```diff
@@ -25,23 +29,23 @@ There are two use cases supported for `match-identities`.
 1. [With gitbase](#use-with-gitbase)
 1. [Without gitbase](#use-without-gitbase)
 
-In both cases, you get a parquet file as a result. 
-Read more in [Output format](#output-format) section.
+In both cases, the output identity table can be either saved as a parquet file or directly to Postgres.
```

Comment:
Actually, we changed this back: we always write the Parquet files. There is a small Python script to push any parquet file to postgres included in this repo